client/blocks/inline-help/popover.jsx

```diff
@@ -132,11 +133,15 @@ class InlineHelpPopover extends Component {
 						onSelect={ this.openResultView }
 						query={ this.props.searchQuery }
 						isVisible={ ! this.state.showSecondaryView }
+						isSearching={ this.props.isSearching }
 					/>
 					<InlineHelpSearchResults
 						onSelect={ this.openResultView }
 						onAdminSectionSelect={ this.setAdminSection }
 						searchQuery={ this.props.searchQuery }
+						isSearching={ this.props.isSearching }
+						searchResults={ this.props.searchResults }
+						hasAPIResults={ this.props.hasAPIResults }
```

Comment:
`InlineHelpSearchResults` is wrapped in `withInlineHelpSearchQuery`, so it already has the `isSearching`, `searchResults` and `hasAPIResults` props. They should either be passed from the parent or injected by the HOC, but not both at the same time.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/55753/commits/7fe38e9be88601b4bd9f1a833c49a33ee24ed4cc